### ISSUE TITLE
[QUESTION] Cannot install Offline and Swap on Android 14?

### ISSUE THREAD

**Issue by NablaDeltaPsi** [PhilippC/keepass2android#2711]
I cannot install K2A Offline and Keyboard Swap on my Galaxy A52s 5G with Android 14. It says in Play Store the app is not available for my smartphone. Why? The normal K2A app works fine.

**Comment by chewi** [comment]
Yeah, I'm a bit confused by this. I still have the Offline version on my Android 13 phone, but my wife's Android 13 phone says it's incompatible and doesn't show it in the search results.

**Comment by Spiralpine** [comment]
I don't think the offline version is supported for the new installs. Why don't you just use the online version and use the "system file picker" to point to the local database?

**Comment by chewi** [comment]
The point of the offline version is to have a smaller attack surface. You can see from the release history that the offline version didn't always get released when the online version did, which does make sense. Perhaps there just haven't been any relevant changes lately, but it does seem to need a new build for newer Android versions.

**Comment by PhilippC** [comment]
probably caused by using a lower "target sdk version" in the play store release. I am working on updating both apps to target Android 14 (see https://github.com/PhilippC/keepass2android/pull/2761)
